extensions/sql-database-projects/src/controllers/mainController.ts

```diff
@@ -63,7 +63,8 @@ export default class MainController implements vscode.Disposable {
 
 		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.build', async (node: WorkspaceTreeItem) => { return this.projectsController.buildProject(node, false); }));
 		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.buildWithCodeAnalysis', async (node: WorkspaceTreeItem) => { return this.projectsController.buildProject(node, true); }));
-		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.publish', async (node: WorkspaceTreeItem) => { return this.projectsController.publishProject(node); }));
+		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.publish', async (node: WorkspaceTreeItem) => { return this.projectsController.publishProject(node, false); }));
+		this.context.subscriptions.push(vscode.commands.registerCommand('sqlDatabaseProjects.publishPreview', async (node: WorkspaceTreeItem) => { return this.projectsController.publishProject(node, true); }));
```

Comment:
I think it'd be better for both `publish` and `publishDialog` to check the setting here instead of hard-coding it for each command.  That way, if anyone is programmatically calling the commands, they get consistent behavior.

With that implementation, the only reason to have two different vscode commands is to add "(preview)" to the display string.